src/components/OnboardingComponents/SemanticSearch/ArticleDialog.tsx

```diff
@@ -0,0 +1,210 @@
+
+  if (!article) {
+    return <></>;
+  }
+
+  function onDocumentLoadSuccess({numPages: nextNumPages}: {numPages: number}) {
+    setNumPages(nextNumPages);
+    setPageNumber(1); // Reset to first page on new document load
+  }
+
+  return (
+    <Dialog
+      open={open}
+      onClose={onClose}
+      maxWidth='lg'
+      fullWidth
+      scroll='paper'
+      sx={{'& .MuiDialog-paper': {height: '90vh'}}}
+    >
+      <DialogTitle sx={{m: 0, p: 6, pb: 5}}>
+        <Typography variant='h2' sx={{pr: '2rem'}}>
+          {article.title}
+        </Typography>
+        <IconButton
+          aria-label='close'
+          onClick={onClose}
+          sx={{
+            position: 'absolute',
+            right: 8,
+            top: 8,
+            color: (theme) => theme.palette.grey[500],
+          }}
+        >
+          <CloseIcon />
+        </IconButton>
+      </DialogTitle>
+      <DialogContent
+        ref={containerRef}
+        dividers
+        sx={{display: 'flex', flexDirection: 'column', alignItems: 'center', p: 1}}
+      >
+        <Document
+          file={article.hyperlink}
+          onLoadSuccess={onDocumentLoadSuccess}
+          loading={<CircularProgress />}
+          error='Failed to load PDF file.'
+        >
+          <Page pageNumber={pageNumber} width={containerWidth > 0 ? containerWidth : undefined} scale={zoom} />
+        </Document>
+      </DialogContent>
+      <Box sx={{display: 'flex', justifyContent: 'center', alignItems: 'center', p: 1, gap: 2}}>
+        <IconButton onClick={goToPreviousPage} disabled={pageNumber <= 1} aria-label='previous page'>
+          <ArrowBackIosNewIcon />
+        </IconButton>
+        <Typography>
+          {pageNumber} / {numPages}
+        </Typography>
+        <IconButton onClick={goToNextPage} disabled={pageNumber >= numPages} aria-label='next page'>
+          <ArrowForwardIosIcon />
+        </IconButton>
+        <Box sx={{display: 'flex', alignItems: 'center', gap: 1, ml: 2}}>
+          <Tooltip title='Zoom Out (Ctrl/Cmd + -)'>
+            <IconButton onClick={handleZoomOut} disabled={zoom <= 0.25} aria-label='zoom out'>
+              <ZoomOutIcon />
+            </IconButton>
+          </Tooltip>
+          <Typography variant='body2' sx={{minWidth: '3rem', textAlign: 'center'}}>
+            {Math.round(zoom * 100)}%
+          </Typography>
+          <Tooltip title='Zoom In (Ctrl/Cmd + +)'>
+            <IconButton onClick={handleZoomIn} disabled={zoom >= 3} aria-label='zoom in'>
+              <ZoomInIcon />
+            </IconButton>
+          </Tooltip>
+          <Tooltip title='Reset Zoom (Ctrl/Cmd + 0)'>
+            <IconButton onClick={handleZoomReset} disabled={zoom === 1} aria-label='reset zoom'>
+              <Typography variant='body2' sx={{fontWeight: 'bold'}}>
+                100%
+              </Typography>
+            </IconButton>
+          </Tooltip>
+        </Box>
+      </Box>
+    </Dialog>
+  );
+}
```

Comment:
Same as above, for sx props please format the to one per line regardless of what the formatter says it's always easier to read.
Also we prefer to use the full keywords (instead of p) so it's easier to understand.
```suggestion
        sx={{
          display: 'flex',
          flexDirection: 'column',
          alignItems: 'center',
          padding: 1
        }}
```

Reply:
Done